public/locales/en/panel-skybrowser.json

```
@@ -0,0 +1,116 @@
+{
+  "add-browser": {
+    "label": "Add Browser",
+    "description": "Powered by WorldWide Telescope"
+  },
+  "wwt": {
+    "info-overlay": {
+      "loading-wwt": "Loading WorldWide Telescope...",
+      "loading-image-collection": "Loading image collection...",
+      "camera-not-in-solar-system": {
+        "title": "Camera is not in the Solar System.",
+        "description": "You need to be in the Solar System to use the SkyBrowser."
+      }
+    },
+    "telescope-view": {
+      "no-skybrowser": "There are no SkyBrowsers. Create on in the SkyBrowser panel.",
+      "no-id": "No browser selected",
+      "drag-area-aria-label": "Dragging area for WorldWide Telescope",
+      "error": "ERROR: cannot display AAS WorldWide Telescope research app!"
+    },
+    "error": {
+      "no-images": "Received 0 AAS WorldWide Telescope images!",
+      "unable-to-load-wwt": "Couldn't load AAS WorldWide Telescope images!",
+      "unable-to-load-images": "Could not load image collection from OpenSpace. Error: {{error}}"
```

```suggestion
      "unable-to-load-images": "Could not load image collection. Error: {{error}}"
```

Users can specify their own collection, so it might have been their own that failed